README.md

```diff
@@ -160,22 +164,22 @@ Now that you are in the notebook, add generated code to insert the data as a Dat
 
 ### 6. Initialize Watson Machine Learning client
 
-The Watson Machine Learning client is required to save and deploy our customer churn predictive model, and should be available on your IBM Cloud Pak for Data platform. Insert the required credentials in the following cell:
+The Watson Machine Learning client is required to save and deploy our customer churn predictive model, and should be available on your IBM Cloud Pak for Data platform. Find the cell containing the code given below and insert the url, username and password for your IBM Cloud Pak for Data instance:
 
   ```python
-  from watson_machine_learning_client import WatsonMachineLearningAPIClient
-
-  # get URL, username and password from your IBM Cloud Pak for Data administrator
-  wml_credentials = {
-    "url": "https://X.X.X.X",
-    "instance_id": "wml_local",
-    "username": "****",
-    "password": "****",
-    "version": "3.0.0"
-  }
-
-  client = WatsonMachineLearningAPIClient(wml_credentials)
-  print(client.version)
+ from ibm_watson_machine_learning import APIClient
+
+# get URL, username and password from your IBM Cloud Pak for Data administrator
+wml_credentials = {
+  "url": "https://X.X.X.X",
+  "username": "*****",
+  "password": "*****",
+  "instance_id": "wml_local",
+  "version" : "3.5"
```

Comment:
if you edit this, delete the extra space between "version" and ":" -- Perhaps not worth mentioning if you arent updating.